### ISSUE TITLE
GTA:SA is capped at 60 FPS

### ISSUE THREAD

**Issue by Gigabitzz** [ThirteenAG/III.VC.SA.WindowedMode#8]
With the modification installed, GTA:SA is capped at 60 FPS. I have a script that can unlock the FPS cap and go beyond 60 FPS yet WindowedMode.ASI is preventing it. Is this an intentional issue?

1920x1080x32 @ Widescreen

**Comment by p0kles** [comment]
This is true, I wish if there was a version that doesnt cap GTA:SA fps at 60

**Comment by kforbro** [comment]
Up

**Comment by baitexx** [comment]
June 28th 2023 and still no solution for this.

**Comment by shadyamr** [comment]
2024 and it isn't resolved.

**Comment by shadyamr** [comment]
@ThirteenAG

**Comment by nathan130200** [comment]
In my case it cap down to 30 fps, its bizarre. Disabled windowed mod running fine at 60 fps or more.

**Comment by VenomousReaperGoat** [comment]
Bump! seems to be capped to 60, probably cuz of vsync? i got a 60hz monitor so i cant confirm it... 90fps without windowed mode.

**Comment by MiranDMC** [comment]
Seems this is how D3D9 Renderware works. It detects that game is windowed and enables VSync, capping FPS at display monitor refresh rate (usually 60Hz).
When using **rwd3d9.dll** for GTA3 and GTA VC it also starts to behave in same way.
Problem needs some investigation and probably hacks in the Renderware itself.
